### ISSUE TITLE
Breaks Moodle 3.1 drag and drop images into Atto

### ISSUE THREAD

**Issue by nadavkav** [cdsmith-umn/pastespecial#15]
When this plugin is enabled and used in Atto (with Moodle 3.1), Images can no longer be dragged and dropped into the Atto editor.

**Comment by cdsmith-umn** [comment]
@nadavkav Thanks for the report. I'll start looking at this right away.

**Comment by cdsmith-umn** [comment]
@nadavkav Can you please describe the steps to reproduce this problem? I have been unable to reproduce it, as of yet.

**Comment by nadavkav** [comment]
While trying to describe the stages, I installed this plugin on a new and clean Moodle 3.1 stable version. and found out it does not conflict directly with a clean Moodle install :smile: 

Although, I still have an issue with the original system about which I opened this issue for. as it has a lot of other plugins installed. and when I disable pastespecial... broken drag and drop of images starts to function again. 
We are on holidays at this time, and funny as it sound... we'll be back in two weeks time. so I am closing this issue for now, as I will have to dig deeper into this and find out which plugin conflict with this one.

Thank you for taking the time to check this. sorry for not making a deeper research before opening this issue.
